packages/create-svelte/templates/default/i18n.config.cjs

```diff
@@ -0,0 +1,45 @@
+const locales = require('./locales.json');
```

Comment:
this file should probably have `.js` extension since `svelte.config.cjs` has been renamed to `svelte.config.js`